mdit_py_plugins/tasklists/__init__.py

```diff
@@ -144,8 +145,4 @@ def is_list_item(token):
 
     def starts_with_todo_markdown(token):
         # leading whitespace in a list item is already trimmed off by markdown-it
-        return (
-            token.content.startswith("[ ] ")
-            or token.content.startswith("[x] ")
-            or token.content.startswith("[X] ")
-        )
+        return re.match(r"\[[ xX]][ \t\n\v\f\r]+", token.content)
```

Comment:
Why not use `\s` here, to match whitespace? (see https://docs.python.org/3/library/re.html)

Reply:
We can do that, but then have to set the `re.ASCII` flag, else we match too much. I listed all the chars so that they are easy check the against the GFM spec which does so too: https://github.github.com/gfm/#whitespace-character

Reply:
The CommonMark specification (because that is the reference here!) does the same: https://spec.commonmark.org/0.29/#whitespace-character .

Reply:
😂 can you a link to this white space definition in a comment or the docstring

Reply:
> can you a link to this white space definition in a comment or the docstring

Sure. I think I'll make it a private variable or something.

Reply:
Well, the new CM spec is not fully updated then - it still refers to whitespace in the examples without defining it: https://spec.commonmark.org/0.30/#example-350 .

I was actually thinking why this should not defined as a constant because it has just proven to change over time... .  GFM_WHITESPACE, CM_WHITESPACE.